### DISCUSSION TITLE
Testing ComfyUI_bnb_nf4_fp4_Loaders Discussions!

### DISCUSSION THREAD

**Discussion by silveroxides** [silveroxides/ComfyUI_bnb_nf4_fp4_Loaders#2] [Announcements]
I am opening up discussions just because users seem to want me to. 
I am without any hardware at the moment after s burglary at my home so won't be particularly active. 
I will check issues but I expect users to try and collaborate and make a PR themselves if any extended functionality is highly requested. 
If it is stable and is an improvement, then I will ofc merge it. I will be more active once I eventually have new gear. In the meantime, behave and be nice to each other.

**Comment by ClothingAI** [comment]
Omg, I am sorry for the burglary. Do you know if it was random or .. 
Anyway I hope you still have your knowledge on internet (please update all your codes, disconnect sessions and make new ones (people can use sessions to make it as if they are connected without even needing to insert code or whatever)
Thanks for the discussions section.
When I asked for it I was in a lot of confusion over flux, comfy, and the different verisions of everything (unet, checkpoint, fp16 8 etc), now I figured lot of my questions. But I would gladly share a question or a thought here if I need it. Or answer others if they ask questions. Thanks

**Comment by rwfsmith** [comment]
I've been testing out a FP4 quantized Wan2.1 model versus GGUF. On a 5070 Ti the FP4 quantized version is still a little slower than a Q6 GGUF. Is there something that needs to be changed to get better FP4 performance on Blackwell? It's supposed to be optimized for it, I believe.
